src/linear-algebra.lisp

```diff
@@ -23,18 +23,28 @@
   "A representation of an operator on a quantum state. This will be a unitary square matrix where each dimension is a power-of-two."
   `(simple-array cflonum (,n ,n)))
 
+;;; This is intended to be a shorthand for testing.
+(defun wf (&rest elts)
```

Comment:
Should `(wf)` raise an error? I have seen this elsewhere, where at least a single argument is required.

Reply:
I was divided: Should there be a zero-element wavefunction (i.e., just a phase)?

Reply:
I don't see a reason to allow zero-element wavefunctions.

Reply:
I'm fine allowing _one_-element wavefunctions, but not zero-element ones.

Reply:
fixed DONE AND DONE